develop-docs/sdk/philosophy.mdx

```diff
@@ -1,9 +1,9 @@
 ---
 title: Philosophy
+description: This document sets some general guidance for how to approach SDK development at Sentry. It should aid both internal and external developers to understand what motivations go into the design of the SDKs and why we're deciding in certain ways.
```

Comment:
```suggestion
description: This document provides general guidance for SDK development at Sentry. It aims to help both internal and external developers understand the motivations behind SDK design decisions and our decision-making process.
```

I know this is not about content but the description didn't read well

Reply:
Thanks for the suggestion! I'll add it ^^